kotest-assertions/src/commonMain/kotlin/io/kotest/properties/Gen.kt

```diff
@@ -137,6 +138,10 @@ interface Gen<T> {
       override fun random(seed: Long?): Sequence<T> = outer.random(seed) + gen.random(seed)
     }
   }
+
+  fun getRandomFor(aSeed:Long?):Random {
```

Comment:
I don't like this as a public function and I don't like this as part of a `Gen`.

How about extracting it to outside of this interface (you can keep it in the same file) and making it internal?

Reply:
@Kerooker Thanks  for feedback, i will make necessary changes